test/integration/viewer_spec.mjs

```diff
@@ -17,10 +17,106 @@ import {
   awaitPromise,
   closePages,
   createPromise,
+  getSpanRectFromText,
   loadAndWait,
 } from "./test_utils.mjs";
 
 describe("PDF viewer", () => {
+  describe("Zoom origin", () => {
+    let pages;
+
+    beforeAll(async () => {
+      pages = await loadAndWait("tracemonkey.pdf", ".textLayer .endOfContent");
```

Comment:
Could we utilize existing viewer functionality to set the desired zoom level, and possibly even the page number, directly here?
```suggestion
      pages = await loadAndWait("tracemonkey.pdf", ".textLayer .endOfContent", "page-width", null, { page: 2 });
```

Reply:
Thanks for the suggestion, it's much simpler now :)